UKXattrMetadataStore.m

```diff
@@ -103,11 +93,24 @@ +(void)	setObject: (id)obj forKey: (NSString*)key atPath: (NSString*)path traver
 								errorDescription: &errMsg];
 	if( errMsg )
 	{
-		[errMsg autorelease];
 		[NSException raise: @"UKXattrMetastoreCantSerialize" format: @"%@", errMsg];
 	}
 	else
-		[[self class] setData: plistData forKey: key atPath: path traverseLink: travLnk];
+		[self setData: plistData forKey: key atPath: path traverseLink: travLnk error: NULL];
+}
+
++(BOOL)	setObject: (id)obj forKey: (NSString*)key atPath: (NSString*)path traverseLink:(BOOL)travLnk error:(NSError**)error
```

Comment:
I'd rename this to `setPlistXML:`. `setObject:` implies it might be the right thing to use for an `NSDate` or `NSString`.

Reply:
(Nevermind, this was my naming mistake from earlier on -- will fix this and leave a compatibility method for ObjC)